src/lib/coda_metrics/coda_metrics.ml

```diff
@@ -321,6 +321,14 @@ module Snark_work = struct
        after every block"
     in
     Gauge.v "scan_state_snark_work" ~help ~namespace ~subsystem
+
+  module Snark_fee_histogram = Histogram (struct
+    let spec = Histogram_spec.of_linear 0. 1. 10
+  end)
```

Comment:
I think we should have a summary instead of a histogram. Looking at the documentation [here](https://prometheus.io/docs/practices/histograms/#quantiles), using a summary allows us to not have to set what the buckets are ahead of time. Since we don't know what the prevailing fees will be like that'd be better.

If we do use a histogram, this spec is wrong: it creates 10 buckets between zero and one, but fees are integers that are often more than one.